app/scripts/app.js

```diff
@@ -197,7 +242,23 @@ angular
 		function($firebaseObject, $firebaseArray) {
 			return new DataRepositoryFactory($firebaseObject, $firebaseArray);
 		}
-	])
+    ])
+    .factory('CollectionService', [
+      function() {
+        return new CollectionService();
+      }
+    ])
+    .constant('AvailableTypes', {
+      Brands: 'brands',
+      Categories: 'categories',
+      Products: 'products'
+    })
+    .constant('ProductSuggestionStatuses', {
+      pending: 'pending', // Created and waiting for validation code
+      inReview: 'in review', // Created and validated; being reviewed by an admin
```

Comment:
Is `in review` meant to be a string or a key? I'm wondering if it's better/safer to always use single words for something like this?

Reply:
Hi @aashnisshah , yes it was meant to be a string, and it's what's displayed on screen.
I can change it to a single word, but I'll have to create a mapper to map it to the display value (which isn't a problem, of course).

Reply:
I've done this now.